Kickstarter-iOS/Views/Controllers/DiscoveryPageViewController.swift

```diff
@@ -277,10 +279,22 @@ internal final class DiscoveryPageViewController: UITableViewController {
     guard let navigator = self.presentedViewController as? ProjectNavigatorViewController else { return }
     navigator.updatePlaylist(playlist)
   }
+
+  @objc private func pulledToRefresh() {
+    self.viewModel.inputs.pulledToRefresh()
+  }
+
+  private func performClosureAnimated(_ animated: Bool, closure: () -> Void) {
```

Comment:
This has no dependency on the instance of `DiscoveryPageViewController` .. could we extract it outside of the class (at the bottom of this file)..then instead of calling it with `self?.` like

```
self?.performClosureAnimated(...)
```
we can simply do
```
performClosureAnimated(...)
```